.gitlab-ci.yml

```diff
@@ -48,38 +49,51 @@ stages: # ---------------------------------------------------------------------
       - ./emData/
     expire_in: 1 week
 
-.job_template: &template_topTF-sim_synth
+.job_template: &template_topTF-sim
   <<: *template_base
-  stage: topTF-sim_synth
+  stage: topTF-sim
   script:
-    - cd IntegrationTests/${PROJ_NAME}
+    - cd IntegrationTests/${PROJ_NAME}/script
     - pwd; ls -la; #debug
-    - mkdir tb/data # Output folder for VHDL TextIO
-    - vivado -mode tcl -source script/generate_project.tcl -tclargs ${PROJ_NAME}
-    - md5sum ./tb/data/FM_L1L2_L3PHIC.txt; md5sum ./tb/data/FM_L5L6_L3PHIC.txt # Output check right after sim
-    - vivado -mode tcl -source ../common/script/synth.tcl -tclargs ${PROJ_NAME}
-    - md5sum ./tb/data/FM_L1L2_L3PHIC.txt; md5sum ./tb/data/FM_L5L6_L3PHIC.txt # Output check repeat at the end for convenience
+    - vivado -mode tcl -source ./makeProject.tcl
+    - vivado -mode tcl -source ./runSim.tcl
+    - md5sum ./dataOut/FM_L1L2_L3PHIC.txt; md5sum ./dataOut/FM_L5L6_L3PHIC.txt # Output check right after sim
+    - md5sum ./dataOut/FM_L1L2_L3PHIC.txt; md5sum ./dataOut/FM_L5L6_L3PHIC.txt # Output check repeat at the end for convenience
```

Comment:
We do not this line anymore because the synth is done in another job

Reply:
Are you saying that the two lines using md5sum can be deleted? I haven't understood their purpose. I just copied them from HDLTopSim.

Reply:
The md5sum check was intended to compare the results of the top.vhd (hand-coded and generated) files in different runs. I guess we can delete both lines since we have the fancy Python script now that gives you much more information. Let me do that